bindings/core/src/method/utils.rs

```diff
@@ -97,27 +100,41 @@ pub enum UtilsMethod {
         /// Output ID
         output_id: OutputId,
     },
+    /// Computes the Foundry ID
+    #[serde(rename_all = "camelCase")]
+    ComputeFoundryId {
+        /// Alias id
+        alias_id: AliasId,
+        /// Serial number
+        serial_number: u32,
+        /// Token scheme kind
+        token_scheme_kind: u8,
+    },
     /// Computes the NFT ID
     #[serde(rename_all = "camelCase")]
     ComputeNftId {
         /// Output ID
         output_id: OutputId,
     },
-    /// Computes the Foundry ID
+    /// Returns the output ID from transaction id and output index
+    ComputeOutputId { id: TransactionId, index: u16 },
+    /// Constructs a tokenId from the aliasId, serial number and token scheme type.
     #[serde(rename_all = "camelCase")]
-    ComputeFoundryId {
-        /// Alias address
-        alias_address: AliasAddress,
-        /// Serial number
+    ComputeTokenId {
+        alias_id: AliasId,
         serial_number: u32,
-        /// Token scheme kind
-        token_scheme_kind: u8,
+        token_scheme_type: TokenScheme,
     },
     /// Compute the hash of a transaction essence.
     HashTransactionEssence {
         /// The transaction essence
         essence: TransactionEssenceDto,
     },
+    /// Calculate the input commitment from the output objects that are used as inputs to fund the transaction.
```

Comment:
We use a mixture of compute and calculate, can you harmonise to Compute since this is the one used in the names?